src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+class Rectangle extends Figure {
+    private int rightSide;
+    private int topSide;
+
+    public Rectangle(String color, int topSide, int rightSide) {
```

Comment:
Avoid passing raw expressions in the constructor. It's better to create separate variables for 'topSide' and 'rightSide' and pass them to the constructor for better code readability.